### DISCUSSION TITLE
Is there anyway to duplicate a library context?

### DISCUSSION THREAD

**Discussion by afshinpir** [openssl/openssl#24620] [Q&A]
Hi all

Is there any way to duplicate a library context? For example, you have main library context `libctx` and you want to duplicate it in order to load another provider into duplicated context. Any idea?

Best Regards

**Comment by paulidale** [comment]
No.

**Comment by t8m** [comment]
However it would IMO be a valid feature request. Not sure how simple it would be to implement it though. Probably not.

**Reply by afshinpir** [reply]
I somehow was thinking maybe we could do it using `OSSL_LIB_CTX_new_child`, but since dispatch and core is not exported, it is not feasible as is.
I will add this as a feature request.

**Reply by paulidale** [reply]
`OSSL_LIB_CTX_new_child` is very different.  You **really** don't want to go there.
A feature request is the way to go.
